assets/js/admin/orders.js

```diff
@@ -11,4 +11,91 @@ jQuery( document ).ready( ( $ ) => {
 
 	'use strict';
 
+	const isCommerceOrder = Boolean( wc_facebook_commerce_orders.is_commerce_order );
+
+	window.WCFacebookCommerceOrderOperations = {
+		/**
+		 * Restrict order status options to only allowed options.
+		 *
+		 * @param {Object} $orderStatus Order select jQuery DOM object
+		 */
+		restrict_order_statuses             : $orderStatus => {
+
+			$orderStatus.find( 'option' ).each( function ( index, option ) {
+
+				// check if option value in the allowed list or not
+				if ( wc_facebook_commerce_orders.allowed_commerce_statuses.indexOf( option.value ) === -1 ) {
+					// delete/remove option if not allowed
+					option.remove();
+				}
+			} );
+		},
```

Comment:
```suggestion
		},
		
		
```

We usually use the same spacing rules as we use for PHP, so double lines between methods.